pkg/cloud/cloud_metadata.go

```diff
@@ -35,7 +35,9 @@ func GetCloudMetadata(ctx context.Context, workspaceDatabaseString, token string
 		cloudWorkspace, _, err = client.OrgWorkspaces.Get(ctx, identityHandle, workspaceHandle).Execute()
 	}
 
-	if err != nil {
+	if err != nil && err.Error() == "404 Not Found" {
```

Comment:
can we put an function in for these - where do we have similar functions - error_helpers?